tools/api-markdown-documenter/src/api-item-transforms/TsdocNodeTransforms.ts

```diff
@@ -166,28 +168,21 @@ function transformTsdocSectionContent(
 function transformTsdocParagraph(
 	node: DocParagraph,
 	options: TsdocNodeTransformOptions,
-): ParagraphNode | undefined {
+): (ParagraphNode | ListNode)[] {
 	// TODO: HTML contents come in as a start tag, followed by the content, followed by an end tag, rather than something with hierarchy.
 	// To ensure we map the content correctly, we should scan the child list for matching open/close tags,
 	// and map the subsequence to an "html" node.
 
+	// Trim leading and trailing line breaks, which are redundant in the context of a paragraph.
+	const children = node.nodes;
+	const adjustedChildren = trimLeadingAndTrailingLineBreaks(children);
```

Comment:
is `const children` is only used in line 178? if so, maybe just remove and do `const adjustedChildren = trimLeadingAndTrailingLineBreaks(node.nodes);`